ScheduleLesson/Exception/ExceptionHandlingMiddleware.cs

```diff
@@ -4,12 +4,13 @@ public class ExceptionHandlingMiddleware
     {
         private readonly RequestDelegate _next;
         private readonly ILogger<ExceptionHandlingMiddleware> _logger;
+        private const int StatusCodeInternalServerError = 500;
```

Comment:
You create 2 the same consts in different files.  
This violates the DRY principle (don't repeat yourself
Find and use ready-made constants from some libs or made public consts in the separate file